README.md

```diff
@@ -91,3 +91,5 @@ When you want to update your Discord client to the latest version of this fork,
 - **[Become Catgirl](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/becomeCatgirl>):** Turns your messages into a catgirl message.
 - **[Talk In Reverse](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/talkInReverse):** Reverses the message content before sending it.
 - **[God Mode](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/godMode):** Get all permissions (client-side).
+- **[Copy Emoji as Formatted String](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/CopyEmojiAsFormattedString)** Adds a button to copy an emoji as a formatted string!
+- **[Custom App Icons](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/CustomAppIcons)** Gives the ability to upload custom (In-)App Icons.
```

Comment:
Use the original plugin names 😭
```suggestion
- **[CopyEmojiAsFormattedString](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/CopyEmojiAsFormattedString)** Adds a button to copy an emoji as a formatted string!
- **[CustomAppIcons](https://github.com/Tolga1452/Vencord/tree/main/src/plugins/CustomAppIcons)** Gives the ability to upload custom (In-)App Icons.
```

Reply:
It's confusing that the other plugins have spaces, but these not. I made a pull request last night to make the other plugins without spaces, but it was denied.

Reply:
> It's confusing that the other plugins have spaces, but these not. I made a pull request last night to make the other plugins without spaces, but it was denied.

Plugin names are personal choices and not confusing at all. Just copy the original plugin name in this case